### PR TITLE
Get Transfer: return real object, not a map

### PR DESCRIPTION
**Pull request by kushal256**

### CODE REVIEW

File: bitgo-java-api/src/main/java/com/bitso/bitgo/v2/BitGoClient.java [hunk 1]
```diff
@@ -66,11 +67,11 @@
      *
      * @param coin
      * @param walletId
-     * @param walletTransferId
+     * @param txid
      * @return
      * @throws IOException
      */
-    Optional<Map<String, Object>> getWalletTransferId(String coin, String walletId, String walletTransferId) throws IOException;
+    Transfer getWalletTransferId(String coin, String walletId, String txid) throws IOException;
```
Comment (chochos) [inline bitgo-java-api/src/main/java/com/bitso/bitgo/v2/BitGoClient.java]: Shouldn't this be an Optional&lt;Transfer&gt;?

**Review by chochos — COMMENTED** [review]
The change of class shouldn't change the fact that it was an Optional

File: bitgo-java-api/src/main/java/com/bitso/bitgo/v2/BitGoClient.java (hunk 1 above)
Comment (kushal256) [reply bitgo-java-api/src/main/java/com/bitso/bitgo/v2/BitGoClient.java]: done

File: bitgo-java-api/src/test/java/com/bitso/bitgo/util/TestClientV2.java [hunk 2]
```diff
@@ -134,7 +135,18 @@ public void listWalletTransfers() throws IOException {
 //            resp.getTransfers().remove(0);
 //        }
         System.out.println("resp json = " + SerializationUtil.mapper.writeValueAsString(resp));
+    }
 
+    @Test
+    public void getWalletTransfers() throws IOException {
+//        try {
```
Comment (charlycrs) [inline bitgo-java-api/src/test/java/com/bitso/bitgo/util/TestClientV2.java]: Maybe lose this if no more tests are being done with the getWalletTransferId

**Review by charlycrs — APPROVED** [review]
lgtm

File: bitgo-java-api/src/test/java/com/bitso/bitgo/util/TestClientV2.java (hunk 2 above)
Comment (kushal256) [reply bitgo-java-api/src/test/java/com/bitso/bitgo/util/TestClientV2.java]: done

**Pull request merged**